src/content/blog/windows-ddev-setup.md

```diff
@@ -40,7 +40,7 @@ Two recent Windows machines I set up were the new ARM64/Qualcomm/CoPilot variety
     * On ARM64, `choco uninstall -y mkcert gsudo` so that the DDEV installer can get the native versions of each of these.
     * On ARM64, install the Windows-side DDEV from the installer in the [DDEV releases](https://github.com/ddev/ddev/releases). We'll probably discontinue documenting the Chocolatey install technique in the future.)
 11. Install and test the fantastic [1Password ssh agent](https://developer.1password.com/docs/ssh/agent/).
-12. On Windows PowerShell `ssh git@github.com git@github.com` to verify that the 1Password SSH agent is working. If it says "PTY Allocation Failed", just hit `<RETURN>` and ignore it. You should get the confirmation message from GitHub.
+12. On Windows PowerShell `ssh -T git@github.com` to verify that the 1Password SSH agent is working. If it says "PTY Allocation Failed", just hit `<RETURN>` and ignore it. You should get the confirmation message from GitHub.
```

Comment:
What does the `-T` do?

Reply:
Can seem to find much but https://explainshell.com/explain?cmd=ssh%20-t says:

> -T Force pseudo-tty allocation.

https://stackoverflow.com/questions/42505339/why-use-t-with-ssh has some more explaination.

Without the `-T`, the command runs alot slower and I get an `PTY allocation request failed on channel 0` error.

IIRC, most SSH debugging tutorials including `-T`.